app/views/organizations/adoptable_pets/index.html.erb

```diff
@@ -12,6 +12,50 @@
 <!--pet tiles-->
 <section class="pb-5" id="adoptable_pets">
   <div class="container">
+    <div class="row">
+      <div class="col m-2">
+      <!--filter section-->
+        <%= search_form_for @q, url:adoptable_pets_path do |f| %>
+          <div class='d-flex justify-content-center align-items-center flex-wrap mb-3'>
```

Comment:
Lets add a few bootstrap classes here to make the filter div look more in line with the screenshot in the issue:
```html
          <div class='d-flex justify-content-center align-items-center flex-wrap mb-3 p-4 rounded-5 bg-white'>
```